packages/paste-website/src/components/shortcodes/component-header/index.tsx

```diff
@@ -7,11 +7,16 @@ import {SidebarCategoryRoutes} from '../../../constants';
 import {P} from '../../Typography';
 import {Heading} from '../../Heading';
 
-const ComponentHeaderBasic: React.FC<{name: string}> = ({name}) => (
+const ComponentHeaderBasic: React.FC<{
+  name: string;
```

Comment:
Just a question. Why set types here over using `interface`?

Reply:
So a couple thoughts:
1- The interface doesn't get re-used, so I just define it inline.
2- Naming interfaces takes effort, and the name is somewhat meaningless here.  I equate this to using the `css` prop rather than naming and creating a styled component.

Reply:
👍 thanks for the explanation. Appreciate it.